src/objects/ObjectContainer.c

```diff
@@ -82,8 +82,14 @@ typedef struct {
     void * arg;
 } StrCmpCtx;
 
-static int ObjectContainerElementCmp(const void * first, const void * second, void * ctx) {
-    StrCmpCtx * data = (StrCmpCtx *)ctx;
+#if(__APPLE__)
+static int ObjectContainerElementCmp(void *ctx, const void *first,
+                                     const void *second) {
+#else
+static int ObjectContainerElementCmp(const void * first, const void * second,
+		                             void * ctx) {
```

Comment:
The indentation contains spaces and tabs. Please just use spaces.

Reply:
Totally right. Messed up my editor configs. Corrected it. Thank you.